pkg/providers/tinkerbell/comparator.go

```diff
@@ -0,0 +1,51 @@
+package tinkerbell
```

Comment:
Are we merging the CAPI comparator changes in this PR?

Reply:
yes